converter/src/main/java/gov/cms/qpp/conversion/model/TemplateId.java

```diff
@@ -95,6 +102,7 @@ public String toString() {
 	TemplateId(String root, Extension extension) {
 		this.root = root;
 		this.extension = extension;
+		this.alwaysStrict = this.name().equals("CLINICAL_DOCUMENT");
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Move the "CLINICAL_DOCUMENT" string literal on the left side of this string comparison. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS1132)